libsolidity/interface/CompilerStack.cpp

```diff
@@ -989,6 +989,7 @@ string CompilerStack::createMetadata(Contract const& _contract) const
 	meta["version"] = 1;
 	meta["language"] = "Solidity";
 	meta["compiler"]["version"] = VersionStringStrict;
+	meta["useLiteralContent"] = m_metadataLiteralSources;
```

Comment:
This should go under `settings`.

Reply:
Just `settings`, or `metadata.settings`?

Reply:
Also needs to go into documentation.

Reply:
oh, nevermind, we _are_ in `metadata`